server/rating_service/rating_service.py

```diff
@@ -381,6 +391,31 @@ def _update_player_object(
         )
         self._player_service_callback(player_id, rating_type, new_rating)
 
+    async def _publish_rating_changes(
+        self,
+        rating_type: str,
+        new_ratings: Dict[PlayerID, Rating],
+        outcomes: Dict[PlayerID, GameOutcome],
+    ):
+        for player_id, new_rating in new_ratings.items():
+            if player_id not in outcomes:
+                self._logger.error("Missing outcome for player %i", player_id)
+                continue
+
+            rating_change_dict = {
+                "player_id": player_id,
+                "rating_type": rating_type,
+                "new_rating_mean": new_rating.mu,
+                "new_rating_deviation": new_rating.sigma,
```

Comment:
Maybe we should also publish the old rating. The league service doesn't need it, but it could be useful for other things in the future.

Reply:
Good point